### PR TITLE
Very very safe match

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Adds a category of matching causatives and managed variants from the case (selectable) default gene panels only.

Screenshot 2023-03-28 at 11 45 26

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 1]
```diff
@@ -563,13 +563,13 @@ <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Pinned variants ({{suspec
   </div>
 {% endmacro %}
 
-{% macro matching_causatives(other_causatives, institute, case) %}
+{% macro matching_causatives(other_causatives, institute, case, default=False) %}
   <div class="card panel-default" >
     <div data-bs-toggle='tooltip' class="panel-heading" title="If there are any variants in this case
-      that have been marked as causative in another case for this insitute">
-      <strong><a data-bs-toggle="collapse" href="#matchingCausatives" class="text-body">Matching causatives from other cases ({{other_causatives|length}})</a></strong>
+      that have been marked as causative in another case for this insitute. {% if default %}Variants in default panels for the case only.{% endif %}">
```
[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/utils.html:569]: Yes, it crossed my mind, but then there is the issue of the whole institute having to agree on how it is done. The current way was easy to implement and fixes the immediate issue.

Another way would be to have a case level switch for it, like next to the "Safe matching" indicator, where one can pick the filtering of matches on a case by case basis.

**Review by northwestwitch — APPROVED** [review]
Works fine so I'm approving. I would modify a bit the layout to make the info in those line a bit less invasive. There would be a lot of empty space in those lines anyway:

image

**Comment by dnil** [comment]
> Works fine so I'm approving. I would modify a bit the layout to make the info in those line a bit less invasive. There would be a lot of empty space in those lines anyway:
> 
> image

Quite! They will not always be there, and it seemed a little complicated to shrink them with them being collapses that kind of need to work independently, but I'll give it a little thought!

**Comment by dnil** [comment]
Ok, I removed a few intervening lines. This is what it looks like now, when all are found:
Screenshot 2023-03-31 at 13 33 51

Its not super nice with all the compact similar text, but at least it serves its purpose and does not take up more space than before.

I noticed the checksums on the fontawesome pack was not checking out any more - they probably updated?
Rescuing e.g. this 
Screenshot 2023-03-31 at 13 30 52
back to this
Screenshot 2023-03-31 at 13 31 00

**Pull request merged**